File: CS/11_Conversion/ToEMF.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Pdf;

namespace ToEMF
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //pdf file
            string input = @"..\..\..\..\..\..\Data\Sample4.pdf";

            //open pdf document
            PdfDocument doc = new PdfDocument(input);

            //save to emf files
            for (int i = 0; i < doc.Pages.Count; i++)
            {
                String fileName = String.Format("Sample4-img-{0}.emf", i);
                using (Image image = doc.SaveAsImage(i,Spire.Pdf.Graphics.PdfImageType.Metafile, 300, 300))
                {
                    image.Save(fileName, System.Drawing.Imaging.ImageFormat.Emf);
                    System.Diagnostics.Process.Start(fileName);
                }
            } 
        }
    }
}

File: VB/11_Conversion/ToEMF.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Pdf

Namespace ToEMF
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			'pdf file
			Dim input As String = "..\..\..\..\..\..\Data\Sample4.pdf"

			'open pdf document
			Dim doc As New PdfDocument(input)

			'save to emf files
			For i As Integer = 0 To doc.Pages.Count - 1
				Dim fileName As String = String.Format("Sample4-img-{0}.emf", i)
				Using image As Image = doc.SaveAsImage(i,Spire.Pdf.Graphics.PdfImageType.Metafile, 300, 300)
					image.Save(fileName, System.Drawing.Imaging.ImageFormat.Emf)
					Process.Start(fileName)
				End Using
			Next i
		End Sub
	End Class
End Namespace
